pulp_rpm/app/modelresource.py

```diff
@@ -154,6 +157,10 @@ class UpdateRecordResource(RpmContentResource):
     class Meta:
         model = UpdateRecord
         import_id_fields = model.natural_key_fields()
+        exclude = BaseContentResource.Meta.exclude + (
```

Comment:
Could you explain what's going on here?  I don't have background in this, I'm sure it's fine, I just don't know what it's doing.

Reply:
exclude= lists the fields we do *not* want to export here (because we can't rely on them/they don't make sense 'downstream'). BaseContentResource (which we are a subclass of here) takes care of the fields from Content (pulp_id, pulp_created, pulp_updated). We want to add some new fields, but since UpdateRecordResource.Meta is its own thing and not a subclass of BaseCOntentResource.Meta we have to be explicit here.

We don't need/want to export collections and references here, because the linkages to those entities get made from the 'other side' at import-time.